tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cartesi/sdk:0.9.0
+FROM cryptobughunters/sdk:0.11.1
 
 RUN apt-get update
 RUN apt-get install -y \
```

Comment:
Consider making this image more deterministic with `--snapshot`.

Reply:
I was unable to use `--snapshot` because the SDK is not based on a recent version of Ubuntu.

Reply:
I'll talk to the ones that maintain this `catresi/sdk` :rofl: